<launch>
  <param name="mesh_in_mm" value="False"/>
  <rosparam param="model_bank" subst_value="True">

    [
    <!--Format-->
    <!--
      [name,
      file_descriptor,
      flipped (up-side down in the images wrt to the model),
      symmetric (rotationally symmetric about Z-axis)]
      -->
      [crate,
      $(find sbpl_perception)/data/crate/crate_new.ply,
      false,
      false,
      1,
      0.06,
      1
      ],
    ]
  </rosparam>
</launch>
